src/App.tsx

```diff
@@ -1,20 +1,34 @@
 import Logo from "@assets/logo.png";
+import ActivityList from "@components/Activities/ActivitiesList";
 import DataContainer from "@components/DataContainer/DataContainer";
 import HelloWorld from "@components/HelloWorld/HelloWorld";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 import styles from "./App.module.css";
 
 export default function App() {
   return (
     <main className={styles.main}>
       <img className={styles.logo} alt="React logo" width="400px" src={Logo} />
-      <DataContainer />
-      <div>
-        {`VITE_SOME_ENVIRONMENT_VARIABLE from .env.local: "${
-          import.meta.env.VITE_SOME_ENVIRONMENT_VARIABLE
-        }"`}
-      </div>
-      <HelloWorld msg="Hello EE Practicum Team 2!" />
+      <BrowserRouter>
+        <Routes>
+          <Route
+            path="/"
+            element={
+              <div>
+                <div>
+                  {`VITE_SOME_ENVIRONMENT_VARIABLE from .env.local: "${
+                    import.meta.env.VITE_SOME_ENVIRONMENT_VARIABLE
+                  }"`}
+                </div>
+                <HelloWorld msg="Hello EE Practicum Team 2!" />
+                <DataContainer />
+              </div>
+            }
+          />
+          <Route path="/activities" element={<ActivityList />} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
Nice, Jillian!! I see that activities are displayed on the `/activities` page, just as described in the ticket! 🙌  We didn't talk about this a lot in our meeting, so it's very cool that you picked that up and implemented a router to satisfy the requirement! 💯

Reply:
Thank you!